src/Entity/Post.php

```diff
@@ -48,86 +47,86 @@ class Post
      *
      * @ORM\Id
      * @ORM\GeneratedValue
```

Comment:
I think you are missing `Mapping` here (and other places): `@ORM\Mapping\GeneratedValue`.

Reply:
I do - doesn't make any sense to waste my time here, since the pull request has been closed already.